pom.xml

```diff
@@ -294,6 +294,18 @@
     </dependency>
   </dependencies>
 
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>com.fasterxml.jackson</groupId>
+        <artifactId>jackson-bom</artifactId>
+        <version>${jackson.version}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
```

Comment:
There is currently a dependency on jackson-databind 2.19+ in lib-linked-data-dictionary, this conflicts with spring-boot-dependencies (from spring-boot-starter-parent) which uses the Jackson BOM for 2.18.2. The earliest spring-boot-dependencies that uses a compatible jackson-databind is 3.5.0 (though it doesn't reach the latest Jackson until 4.0.0-M3, the latest pre-release).

Rather than keep this section or otherwise try to change a number of core dependencies in this project, it's probably easiest to change lib-linked-data-dictionary to use an older API call compatible with Jackson 2.18.

@pkjacob

Reply:
After merging the dictionary PR, I removed the dependencyManagement section and the Jackson version, redeployed mod-data-export to edev (clearing the MVN cache along the way), and verified it still works as expected.